src/viam/sdk/spatialmath/orientation.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <viam/api/app/v1/robot.pb.h>
 
+#include "orientation.hpp"
```

Comment:
(nit) just for consistency's sake, can we go with angle brace syntax rather than quotation marks?